miden-lib/asm/miden/sat/internal/prologue.masm

```diff
@@ -656,7 +656,7 @@ end
 #! Process the consumed notes data provided via the advice provider. This involves reading the data
```

Comment:
Somehow it is weird to say `Process the consumed notes` before the notes are being consumed. I would have expected to call those notes `input notes` or `notes to be consumed`

Reply:
Yeah - `input notes` would have been better. We'll move towards this terminology over time.